jenkins/jobs/jobs.yaml

```diff
@@ -19,7 +19,7 @@
           sudo /usr/bin/git remote update
           sudo /usr/bin/git checkout origin/master
           cd -
-          
+
```

Comment:
Hi, Bob, sorry about the formatting changes. I will take care of such kind of things next time.